--- glbinding/CMakeLists.txt	2020-06-26 14:59:00.000000000 +0200
+++ glbinding/CMakeLists.txt.Radium	2020-06-26 15:09:42.000000000 +0200
@@ -136,8 +136,6 @@
 endif()
 
 # Installation paths
-if(UNIX AND SYSTEM_DIR_INSTALL)
-    # Install into the system (/usr/bin or /usr/local/bin)
     set(INSTALL_ROOT      "share/${project}")       # /usr/[local]/share/<project>
     set(INSTALL_CMAKE     "share/${project}/cmake") # /usr/[local]/share/<project>/cmake
     set(INSTALL_EXAMPLES  "share/${project}")       # /usr/[local]/share/<project>
@@ -149,22 +147,7 @@
     set(INSTALL_DOC       "share/doc/${project}")   # /usr/[local]/share/doc/<project>
     set(INSTALL_SHORTCUTS "share/applications")     # /usr/[local]/share/applications
     set(INSTALL_ICONS     "share/pixmaps")          # /usr/[local]/share/pixmaps
-    set(INSTALL_INIT      "/etc/init")              # /etc/init (upstart init scripts)
-else()
-    # Install into local directory
-    set(INSTALL_ROOT      ".")                      # ./
-    set(INSTALL_CMAKE     "cmake")                  # ./cmake
-    set(INSTALL_EXAMPLES  ".")                      # ./
-    set(INSTALL_DATA      ".")                      # ./
-    set(INSTALL_BIN       ".")                      # ./
-    set(INSTALL_SHARED    "lib")                    # ./lib
-    set(INSTALL_LIB       "lib")                    # ./lib
-    set(INSTALL_INCLUDE   "include")                # ./include
-    set(INSTALL_DOC       "doc")                    # ./doc
-    set(INSTALL_SHORTCUTS "misc")                   # ./misc
-    set(INSTALL_ICONS     "misc")                   # ./misc
-    set(INSTALL_INIT      "misc")                   # ./misc
-endif()
+    set(INSTALL_INIT      "etc/init")              # /etc/init (upstart init scripts)
 
 # Set runtime path
 set(CMAKE_SKIP_BUILD_RPATH            FALSE) # Add absolute path to all dependencies for BUILD
